packages/kbn-pm/src/production/build_production_projects.js

```diff
@@ -77,14 +78,26 @@ async function copyToBuild(project, kibanaRoot, buildRoot) {
   // dependencies. The primary reason for allowing the Kibana build process to
   // install the dependencies is that it will "dedupe" them, so we don't include
   // unnecessary copies of dependencies.
-  await copy(['**/*', '!package.json', '!node_modules/**'], buildProjectPath, {
```

Comment:
Nitpick: Feels like we don't need to remove the `!package.json`, but rather try to read it from `project.getCopyBuildDirectory()` below? Then we won't need the `try/catch` either, as it would read the original one if there isn't an intermediate file defined.